set.mm

```diff
@@ -257942,15 +259598,15 @@ irreducible elements (see ~ df-irred ).  (Contributed by Mario Carneiro,
     $d r s v w f x y $.
     $( Define the set of ring homomorphisms from ` r ` to ` s ` .  (Contributed
        by Stefan O'Rear, 7-Mar-2015.) $)
-    df-rnghom $a |- RingHom = ( r e. Ring , s e. Ring |->
+    df-ringhm $a |- RingHom = ( r e. Ring , s e. Ring |->
       [_ ( Base ` r ) / v ]_ [_ ( Base ` s ) / w ]_
    { f e. ( w ^m v ) | ( ( f ` ( 1r ` r ) ) = ( 1r ` s ) /\ A. x e. v A. y e. v
    ( ( f ` ( x ( +g ` r ) y ) ) = ( ( f ` x ) ( +g ` s ) ( f ` y ) ) /\
      ( f ` ( x ( .r ` r ) y ) ) = ( ( f ` x ) ( .r ` s ) ( f ` y ) ) ) ) } ) $.
 
     $( Define the set of ring isomorphisms from ` r ` to ` s ` .  (Contributed
        by Stefan O'Rear, 7-Mar-2015.) $)
-    df-rngiso $a |- RingIso = ( r e. _V , s e. _V |->
+    df-ringim $a |- RingIso = ( r e. _V , s e. _V |->
       { f e. ( r RingHom s ) | `' f e. ( s RingHom r ) } ) $.
```

Comment:
Why not `~df-rim` ? I think `rim` is used elsewhere.

Reply:
Is fixed meanwhile.